### ISSUE TITLE
{Freeway Bug} [Windows Terminal]: 'System Menu' of Windows Terminal application is not adapting to dark mode.

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#20576]
### Windows Terminal version

1.25.1912.0

### Windows build number

28120.2760

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 28120.2760)
App: Windows Terminal Preview

### Steps to reproduce

**Pre-requisite:**
Settings> Personalization> Colors> Choose your Mode> Dark Mode.

**Repro steps:**
1. Open the 'windows terminal' application.
2. Follow the above-mentioned pre-requisite step.
3. Press 'Alt + Space' keys to open system menu.
4. Observe the issue here.

**User experience:**
The low visibility of selected controls in dark mode negatively affects users, especially those with visual impairments or users who rely on dark mode for visual comfort. This issue impairs usability by making it harder to confirm selected options.

**Attachment:**
https://onedrive.cloud.microsoft/:u:/a@pz6xd287/S/IQDL3qbkEjSpRaNVuHU5x_YLAV9OPa0ViEQ-gKebD_ZqFrA?e=D11k1O

### Expected Behavior

The 'System Menu' of 'Windows Terminal' application should adapt to the system’s dark mode settings, providing a consistent visual experience that aligns with user preferences and reduces eye strain in low-light conditions.

### Actual Behavior

'System Menu' of 'Windows Terminal' application does not adapt to the system’s dark mode settings, remaining in light mode regardless of the system preference.

**Comment by DHowett** [comment]
Thanks. This is a duplicate of #20507 and cannot be fixed with public Windows APIs. We will not fix this.
